frontend-react/src/components/ManagePublicKey/ManagePublicKey.tsx

```diff
@@ -1,18 +1,136 @@
-import React from "react";
-import { GridContainer } from "@trussworks/react-uswds";
+import React, { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { GridContainer, Icon, SiteAlert } from "@trussworks/react-uswds";
 
 import { AuthElement } from "../AuthElement";
 import { withCatchAndSuspense } from "../RSErrorBoundary";
+import { USLink } from "../USLink";
+import { showError } from "../AlertNotifications";
 import { MemberType } from "../../hooks/UseOktaMemberships";
+import { validateFileType, validateFileSize } from "../../utils/FileUtils";
+
+import ManagePublicKeyChooseSender from "./ManagePublicKeyChooseSender";
+import ManagePublicKeyUpload from "./ManagePublicKeyUpload";
+
+export const CONTENT_TYPE = "application/x-x509-ca-cert";
+export const FORMAT = "PEM";
+
+function ManagePublicKeySwitchDisplay() {
+    const navigate = useNavigate();
+    const [sender, setSender] = useState("");
+    const [fileContent, setFileContent] = useState("");
+    const [file, setFile] = useState<File | null>(null);
+    const [fileSubmitted, setFileSubmitted] = useState(false);
+
+    // TODO: mocked for now - make the call you need when sending the file
+    const { sendFile } = {
+        sendFile: (data: {
+            contentType?: string;
+            fileContent?: string;
+            file?: File | null;
+        }) => {
+            data = { file: null };
+            return data;
+        },
+    };
+
+    const onSenderSelect = (selectedSender: string) => {
```

Comment:
Minor point, but unless we need to reference callbacks in multiple places, they can just be defined inline like:

```tsx
onSenderSelect={ (selectedSender: string) => setSender(selectedSender) }
```